### ISSUE TITLE
Link from, for e.g., version/0.22/generate, to stable/reference

### ISSUE THREAD

**Issue by dumbledad** [pandas-dev/pandas-website#72]
Google often drops me in specific documentation from a single release, rather than the stable reference. For example the google search [pandas groupby aggregate](https://www.google.com/search?safe=active&rlz=1C1CHBD_en-GBIE841IE842&ei=4eMIXbCFKaXD8gKrzJLoBg&q=pandas+groupby+aggregate&oq=pandas+groupby+aggregat&gs_l=psy-ab.3.0.0i67j0l9.4663.11142..13900...1.0..0.52.506.10......0....1..gws-wiz.......0i71j35i39j0i20i263.BPBJmUg8WD8) had these two as top results:

- https://pandas.pydata.org/pandas-docs/version/0.23/generated/pandas.core.groupby.DataFrameGroupBy.agg.html
- https://pandas.pydata.org/pandas-docs/version/0.22/generated/pandas.core.groupby.DataFrameGroupBy.agg.html

Neither of those pages include an easy way to get back to the main version of the page: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.agg.html

It would be useful if it were easier to get from a specific version of the docs to the stable reference version.

**Comment by jorisvandenbossche** [comment]
Yes, that is a good idea. This is probably something that should be included in the actual documentation (and not the main website), which lives in the main repo: https://github.com/pandas-dev/pandas. Would you want to open an issue there?

**Comment by jorisvandenbossche** [comment]
I remember to have seen docs where there is a bar remembering you this is not the latest documentation, with a link to the latest version of that page. But don't directly know for which project I have seen this.

**Comment by TomAugspurger** [comment]
readthedocs adds one automatically.
